src/dymoprint/dymo_print_engines.py

```diff
@@ -287,6 +287,9 @@ def print_label(
     equal to 1 are burned.
     """
 
+    if detected_device == None:
```

Comment:
```suggestion
    if detected_device is None:
```

Reply:
But it worked! :)
Perhaps it always returned true, I can't tell.